web/main.rb

```diff
@@ -4,31 +4,38 @@
 require 'string-utility'
 require_relative '../utils/utilities'
 
-gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
-demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
-adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
+$gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
+$demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
+$adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
```

Comment:
Do you have a particular reason for making these globals? Globals are generally bad practice and I don't see any reason for making these global instead of local variables. Worst case scenario you would need a module to hold this info, but I don't see the need